language-server/test/dotty/tools/languageserver/WorksheetTest.scala

```diff
@@ -0,0 +1,85 @@
+package dotty.tools.languageserver
+
+import org.junit.Test
+
+import dotty.tools.languageserver.util.Code._
+
+class WorksheetTest {
+
+  @Test def evaluateExpression: Unit = {
```

Comment:
I would add a bunch of test from Scastie: https://github.com/scalacenter/scastie/blob/master/sbt-runner/src/test/scala/com.olegych.scastie.sbt/SbtActorTest.scala

* [ ] multiple `while(true) { }`
* [ ] exceptions `1 / 0`
* [ ] utf-8 encoding
* [ ] System.err
* [ ] initialization issues